modules/ossm-members.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/installing-ossm.adoc
+// * service_mesh/v2x/installing-ossm.adoc
+
+[id="ossm-members_{context}"]
+= Creating the {ProductName} members
+
+`ServiceMeshMember` resources provide a way for {ProductName} administrators to delegate permissions to add projects to a service mesh, even when the respective users don't have direct access to the service mesh project or member roll. While project administrators are automatically given permission to create the `ServiceMeshMember` resource in their project, they cannot point it to any `ServiceMeshControlPlane` until the service mesh administrator explicitly grants access to the service mesh. Administrators can grant users permissions to access the mesh by granting them the `mesh-user` user role. In this example, `istio-system` is the control plane namespace.
+
+----
+$ oc policy add-role-to-user -n istio-system --role-namespace istio-system mesh-user <user_name>.
+----
+
+Administrators can modify the `mesh-user` role binding in the control plane project to specify the users and groups that are granted access. The `ServiceMeshMember` adds the project to the `ServiceMeshMemberRoll` within the control plane project that it references.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+spec:
+  controlPlaneRef:
+    namespace: istio-system
+    name: minimal-install
```

Comment:
line 26 name should be consistent across multiple doc pages. In the doc "Deploying the control plane from the CLI" , the SMCP name: basic. So it's better to update line 26 with "name: basic"